src/js/builtins/JSBufferConstructor.ts

```diff
@@ -29,6 +29,11 @@ export function from(items) {
       }
     }
   }
+  if (typeof items === "object") {
+    if (items.type === "Buffer" && Array.isArray(items.data)) {
+      return new $Buffer(Uint8Array.from(items.data).buffer);
```

Comment:
This creates two Uint8Array.

We don't have to pay that performance overhead.

```suggestion
      return new $Buffer(items.data);
```